transformer_lens/HookedTransformer.py

```diff
@@ -986,6 +986,12 @@ def set_use_attn_result(self, use_attn_result):
         """
         self.cfg.use_attn_result = use_attn_result
 
+    def set_use_attn_input(self, use_attn_input):
```

Comment:
set_use_attn_input_hooks seems like a better name (I should probably change use_attn_result too lol)